### PR TITLE
Fixes Failing Tests on Gecko Master

### PR DESCRIPTION
**Pull request by galenmarchetti**
Changelog describes all the changes here:

```
* Specify --http-host CLI flag in GetStartCommand to have RPC calls bind to publicIP
* Migrate AdminAPI endpoints to the new InfoAPI
* Parse error message on XChain Imports in order to wait for PChain transaction to be accepted
```

### CODE REVIEW

File: commons/ava_networks/high_level_gecko_client.go [hunk 1]
```diff
@@ -274,8 +275,18 @@ func (highLevelGeckoClient HighLevelGeckoClient) TransferAvaPChainToXChain(
 	}
 	// XChain API only accepts the XChain address with the xchain prefix.
 	txnId, err := client.XChainApi().ImportAVA(xchainAddress, username, password)
-	if err != nil {
-		return "", stacktrace.Propagate(err, "Failed import AVA to xchainAddress %s", xchainAddress)
+	for err != nil {
+		/*
+			HACK HACK HACK because the PChain does not have a way to verify transaction acceptence yet,
+			we retry based on the contents of the error message from the XChain call if the pchain transaction
```
Comment (mieubrisse) [inline commons/ava_networks/high_level_gecko_client.go:281]: Noice

File: commons/ava_networks/high_level_gecko_client.go [hunk 2]
```diff
@@ -274,8 +275,18 @@ func (highLevelGeckoClient HighLevelGeckoClient) TransferAvaPChainToXChain(
...
+			has not yet reached consensus
+		*/
+		// TODO TODO TODO When the PChain transaction status endpoint is deployed, use that to wait fro transaction acceptance
+		if strings.Contains(err.Error(), NO_IMPORT_INPUTS_ERROR_STR) {
+			txnId, err = client.XChainApi().ImportAVA(xchainAddress, username, password)
```
Comment (mieubrisse) [inline commons/ava_networks/high_level_gecko_client.go:286]: This probably needs to be in a "retry 5 times, with 1s in between" loop

File: gecko_client/info_api.go [hunk 3]
```diff
@@ -6,15 +6,15 @@ import (
...
 
-type AdminApi struct {
+type InfoApi struct {
 	rpcRequester jsonRpcRequester
 }
 
-func (api AdminApi) GetPeers() ([]Peer, error) {
-	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(adminEndpoint, "admin.peers", make(map[string]interface{}))
+func (api InfoApi) GetPeers() ([]Peer, error) {
+	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(adminEndpoint, "info.peers", make(map[string]interface{}))
```
Comment (mieubrisse) [inline gecko_client/info_api.go:17]: Can we also rename `admin_api_types` to `info_api_types` please?

File: commons/ava_networks/high_level_gecko_client.go (hunk 2 above)
Comment (galenmarchetti) [reply commons/ava_networks/high_level_gecko_client.go:286]: @mieubrisse  right now this retries indefinitely until the test times out, why limit it to 5 retries?

**Pull request merged**